### ISSUE TITLE
Toggle to show all at once *by group*

### ISSUE THREAD

**Issue by AnneCav** [nico3333fr/jquery-accessible-hide-show-aria#26]
Now we have a button to toggle/display all at once, thank you nico. But sometimes, we just want to toggle/display a group of elements, not all of them. Sorry my english is too bad, I switch in french right away :)

On devrait pouvoir "tout" ouvrir/fermer en fonction d'une section/div et pas tous les éléments repliables de toute la page. Typiquement, on imagine une page à deux colonnes : une FAQ en partie centrale et des boîtes de filtres repliables dans une colonne latérale. Je devrais pouvoir ouvrir toute la FAQ sans ouvrir les filtres et vice-versa.

**Comment by nico3333fr** [comment]
Yes, this will be possible when a rewrite that I should do for a long time, in order to have a real plugin, will be done. 

Je garde l'idée, et j'espère vite arriver à le réécrire pour inclure ça de facto ;)

**Comment by AnneCav** [comment]
Apparemment je ne sais pas faire un lien entre une pull request et un ticket.

La pull request est là https://github.com/nico3333fr/jquery-accessible-hide-show-aria/pull/27 :)
